File: client/projects/data/src/lib/models/faq.ts
import { DynamicDocumentType } from '../enums';
import { DynamicDocumentPhoto } from './dynamic-document-photo';

export interface Faq {
  id: number;
  title: string;
  type: DynamicDocumentType;
  introduction: string;
  featuredImage: DynamicDocumentPhoto;
  faqItems: FaqItem[];
}

export interface FaqItem {
  question: string;
  answer: string;
}

File: server/Audi/Models/Faq.cs
using Newtonsoft.Json;

namespace Audi.Models
{
    public class Faqs
    {
        [JsonProperty("faqItems")]
        public FaqItem[] FaqItems { get; set; }
    }
    public class FaqItem
    {
        [JsonProperty("question")]
        public string Question { get; set; }
        [JsonProperty("answer")]
        public string Answer { get; set; }
    }
}